edx_rest_framework_extensions/auth/jwt/middleware.py

```diff
@@ -35,33 +35,34 @@
 log = logging.getLogger(__name__)
 
 
+def _iter_included_base_classes(view_permissions):
```

Comment:
Did this move unchanged? Took a bit to realize that this was not part of the fix.

Reply:
Yep, I moved it out of the class to re-use in a different class, but did not modify the logic.